qutebrowser/javascript/webelem.js

```diff
@@ -102,15 +102,25 @@ window._qutebrowser.webelem = (function() {
         out.attributes = attributes;
 
         const client_rects = elem.getClientRects();
+
+        // Get location of frame and add it to element
+        // TODO How to generate a 0 object without this
+        let frame_offset_rect = null;
+        if (frame === null) {
+            frame_offset_rect = document.head.getBoundingClientRect();
+        } else {
+            frame_offset_rect = frame.getBoundingClientRect();
```

Comment:
`new ClientRect()` throws an error, and I don't think it's worth creating a custom object for this. Is there a better way to create a ClientRect with all 0 values?

Reply:
Do you want something like this?

```js
const defaultRect = {
    top: 0,
    right: 0,
    bottom: 0,
    left: 0,
    height: 0,
    width: 0,
};
```